opentracing/src/main/java/opentracing/Tracer.java

```diff
@@ -36,7 +36,7 @@
   /**
    * Like {@link #startTrace(String)}, but the returned span is made a child of {@code parent}.
    */
-  Span joinTrace(String operationName, TraceContext parent);
+  Span joinTrace(String operationName, SpanContext parent);
```

Comment:
I like SpanContext better than TraceContext, it is scoped more reliably. I think maybe at some point people were thinking about propagating attributes upstream (ex. in http, a *response* header), maybe that led to TraceContext? but still it ends up an attribute of one span.